subprojects/language-jvm/src/testFixtures/groovy/org/gradle/api/tasks/compile/AbstractCompilerContinuousIntegrationTest.groovy

```diff
@@ -17,9 +17,15 @@
 package org.gradle.api.tasks.compile
 
 import org.gradle.launcher.continuous.Java7RequiringContinuousIntegrationTest
+import org.gradle.workers.internal.WorkerDaemonExpiration
 
 
 abstract class AbstractCompilerContinuousIntegrationTest extends Java7RequiringContinuousIntegrationTest {
+
+    def setup() {
+        executer.withArgument "-D${WorkerDaemonExpiration.DISABLE_EXPIRATION_PROPERTY_KEY}=true"
```

Comment:
But yes, when we'll tackle worker daemons reuse across builds, `GradleExecutor#requireIsolatedDaemons()` should apply to both build and worker daemons.